Source/utils/paths.cpp

```diff
@@ -52,6 +52,9 @@ std::string FromSDL(char *s)
 		Log("{}", SDL_GetError());
 		SDL_ClearError();
 	}
+#ifdef defined(__MORPHOS__) || defined(__amigaos__)
+	AddTrailingSlash(result);

```

Comment:
This should be moved to the Amiga implementation of SDL_GetBasePath and SDL_GetPrefPath

Reply:
I took this from Morphos, probably not needed as I've modified SDL_GetBasePath and SDL_GetPrefPath already.